curator/defaults/filter_elements.py

```diff
@@ -105,6 +105,10 @@ def timestring(**kwargs):
     else:
         return { Optional('timestring', default=None): Any(str, unicode, None) }
 
+def threshold_behavior(**kwargs):
+    # This setting is only used with the space filtertype and defaults to 'greater_than'.
+    return { Optional('threshold_behavior', default='larger'): Any('greater_than', 'less_than') }
```

Comment:
This is going to cause a problem:

```
default='larger'
```

This has to be `greater_than` or we'll have serious issues.

Reply:
Good catch. I'm still reviewing my changes :)

Reply:
I should probably have a test for the default behavior